knimin/handlers/pm_plate_map.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from tornado.web import authenticated
+from knimin.handlers.base import BaseHandler
+from knimin import db
+from knimin.handlers.access_decorators import set_access
+from json import dumps
+
+
+@set_access(['Admin'])
+class PMPlateMapHandler(BaseHandler):
+    @authenticated
+    def get(self):
```

Comment:
I think I don't understand this get. It looks like based on a parameter returns completely different info. In my opinion, this should be 4 different handlers, so it behaves like a REST api. Specially, because in one of the cases is returning an HTML, while in the other 3 is returning a JSON. I think this can be confusing for code maintenance.